flink-staging/flink-streaming/flink-streaming-scala/src/test/java/org/apache/flink/streaming/scala/api/CsvOutputFormatITCase.java

```diff
@@ -36,6 +37,10 @@ protected void preSubmit() throws Exception {
 	@Override
 	protected void testProgram() throws Exception {
 		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath);
+		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath, 1);
```

Comment:
If it is expected to fail, could you tell me how to pass the test?
Just I have no idea about this.